### PR TITLE
Fixes small imprecisions in the jupyter notebooks (issue #368)

### PR DESCRIPTION
**Pull request by cattabiani**
- .gitignore ignores jupyter folder .ipynb_checkpoints
- kinetic schemes now correctly displays the law of mass action
- m' = m^3 in cnexp now is correctly computed. Added an
  additional example to show what happens if the equation is not
  handled. This fixes issue #368

### CODE REVIEW

**Comment by cattabiani** [comment]
please retest

**Comment by cattabiani** [comment]
please retest

File: setup.py [hunk 1]
```diff
@@ -10,18 +10,26 @@
...
+"""
+A generic wrapper to access nmodl binaries from a python installation
+Please create a softlink with the binary name to be called.
+"""
+import stat
+from pkg_resources import working_set
+from pywheel.shim.find_libpython import find_libpython
+
 
 # Main source of the version. Dont rename, used by Cmake
```
Comment (pramodk) [inline setup.py:25]: Just to be sure: from the subject topic, is this related change?

Comment (cattabiani) [reply setup.py:25]: this is due to "blacking" setup.py. I just reordered the code around
